docker/dlang

```diff
@@ -25,6 +25,15 @@ apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EBCF975E5BA24D5E
 wget http://master.dl.sourceforge.net/project/d-apt/files/d-apt.list \
                -O /etc/apt/sources.list.d/d-apt.list
 
+if test "$(echo "$VERSION_DMD" | cut -d. -f1)" -eq 2 -a \
+	"$(echo "$VERSION_DMD" | cut -d. -f2)" -ge 077
```

Comment:
Sanity check: is `077` in octal ? :-)

Reply:
It seems it isn't.

Reply:
Exactly [my question](https://github.com/sociomantic-tsunami/beaver/pull/37#discussion_r156143595), I think we shouldn't have leading zeros even if it's correct because it proved to be confusing. Now, just because it's proven in beaver and it is a temporary hack anyway, I think I would rather leave it as it is.